### PR TITLE
chore: Adjust name of acceptance test runner workflow

### PR DESCRIPTION
**Pull request by AgustinBettati**
## Description

Followup from [CLOUDP-215164](https://jira.mongodb.org/browse/CLOUDP-215164)

Avoid having the same name in our actions UI:
Screenshot 2024-01-08 at 10 19 42

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.

### CODE REVIEW

File: .github/workflows/acceptance-tests-runner.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: 'Acceptance Tests'
+name: 'Acceptance Tests Execution'
```
Comment (andreaangiolillo) [inline .github/workflows/acceptance-tests-runner.yml:1]: Could we add a [description](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#description) to our actions? I see how new joiners might be confused by their names/purpose

**Review by lantoli — APPROVED** [review]

File: .github/workflows/acceptance-tests-runner.yml (hunk 1 above)
Comment (AgustinBettati) [reply .github/workflows/acceptance-tests-runner.yml:1]: good point, unfortunately workflow does not support top-level descriptions :/ (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions)
Moved a a comment clarifying the purpose to the top.

**Review by oarbusi — APPROVED** [review]

File: .github/workflows/acceptance-tests-runner.yml (hunk 1 above)
Comment (andreaangiolillo) [reply .github/workflows/acceptance-tests-runner.yml:1]: Thanks! Do you mind adding a comment to other active actions as well or creating a ticket to do so?

**Review by marcosuma — APPROVED** [review]

File: .github/workflows/acceptance-tests-runner.yml (hunk 1 above)
Comment (AgustinBettati) [reply .github/workflows/acceptance-tests-runner.yml:1]: Included 👍

**Review by andreaangiolillo — APPROVED** [review]
LGTM. Thanks for all the follow up : )

**Pull request merged**